client/js/render.js

```diff
@@ -101,6 +101,20 @@ function buildChatMessage(msg) {
 		template = "msg_unhandled";
 	}
 
+	// Make the MOTDs a little nicer if possible
+	if (msg.type === "motd") {
+		let lines = msg.text.split("\n");
+
+		// If all non-empty lines of the MOTD start with a hyphen (which is common
+		// across MOTDs), remove all the leading hyphens.
+		if (lines.every((line) => line === "" || line === "-" || line.startsWith("- "))) {
+			lines = lines.map((line) => line.substr(2));
+		}
+
+		// Remove empty lines around the MOTD (but not within it)
+		msg.text = lines.join("\n").trim();
```

Comment:
I put it there specifically because, even if we can't detect the MOTD as all-hyphen-leading one, we should still try to trim extra newlines before/after the content.

Reply:
Can't you trim before splitting it?

Reply:
I still want to trim after removing the dashes, for example when there are extra lines at the end with just a dash